packages/docker-gc/extra/dcos-docker-gc.service

```diff
@@ -1,16 +1,10 @@
 [Unit]
 Description=Docker GC: periodically garbage collects Docker containers and images
 ConditionPathExists=/opt/mesosphere/etc/docker_gc_enabled
+
 [Service]
 Type=simple
-Restart=on-failure
-RestartSec=20
-LimitNOFILE=16384
-User=dcos_docker_gc
-EnvironmentFile=/opt/mesosphere/environment
-EnvironmentFile=-/var/lib/dcos/docker-gc.env
-Environment=STATE_DIR=/var/lib/dcos/docker-gc
-Environment=PID_DIR=/var/lib/dcos/docker-gc
-Environment=LOG_TO_SYSLOG=0
-Environment=REMOVE_VOLUMES=1
-ExecStart=$PKG_PATH/docker-gc
+ExecStart=/bin/sh -c 'docker system prune --volumes --force'
```

Comment:
This would prune all the stopped containers, even those stopped just a second ago. `docker-gc` only removed old containers (1h?). 

You might want to have a look at https://github.com/spotify/docker-gc/blob/539c30b823f01278a968f6c360c5963e75396f25/docker-gc#L22-L25

Reply:
Yes I know about this but the only solution to this problem is to create a script and manually filter images by container exit time. I think it's OK to remove everything. At least it's clean and we do not any additional logic. I understand that this might increase startup time of failed container but this is the trade off we need to make to keep things simple.

Reply:
`dcoker system prune`  removes only "unreferenced-dangling" images not all old images. So it should be safe option. If container if flaking then it probably will not be deleted since it will be still in use.